src/main/java/com/dj/models/mnist/MnistTrainer.java

```diff
@@ -27,17 +34,15 @@
 
 public class MnistTrainer {
 
-    public static void trainMnistNN(final boolean debug) {
+    private static final int INDEX_OF_KAGGLE_IMAGES = 0;
+    private static final int INDEX_OF_KAGGLE_LABELS = 1;
+
+    public static void downloadDataAndtrainMnistNN(final boolean debug) {
         final ModelWrapper modelWrapper = createTheModel(debug);
-        trainMnistNN(modelWrapper);
+        downloadDataAndtrainMnistNN(modelWrapper);
```

Comment:
@b0noI ...AndTrain... - T is a capital letter.

Reply:
done